docs/test_runners.md

```diff
@@ -0,0 +1,34 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+Debe ser asíncrono para ahorrar tiempo ejecutando tests y complejidad del `framework`.
```

Comment:
Los criterios deben ser también medibles para que se pueda distinguir uno de otro. Aparte, no sé qué significa "asíncrono". Es una herramienta de línea de órdenes. Todos o ninguno lo van a ser.
De veras, intentadlo un poco para elegir *de verdad* una orden que os ejecute los tests, en vez de cambiar lo que decís aquí para justificar la elección que habéis hecho desde el principio

Reply:
Acabo de darme cuenta de que en alguno de los cambios de criterios borré el valor que `snyk` le daba a las tecnologías que estaba comparando, las voy a volver a añadir, ya que es un criterio medible y es importante